test_collections/sdk_tests/support/python_testing/models/python_test_folder.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from pathlib import Path
+
+from test_collections.sdk_tests.support.paths import SDK_CHECKOUT_PATH
+
+UNKNOWN_version = "Unknown"
+VERSION_FILE_FILENAME = ".version"
+
+
+class PythonTestFolder:
```

Comment:
This class looks almost like an exact copy of YamlTestFolder, except the file `python_file_paths` use `.py` instead of `.yaml`

Could we reuse this?

Reply:
Make sense. I'll do this.